scripts/release/publish-docs-artifacts.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Publish material2 docs assets to the material2-docs-content repo
+# material.angular.io will pull from this assets repo to get the latest docs
+
+cd "$(dirname $0)/../../"
+
+docsPath="./dist/docs"
+tempRepoPath="./dist/tempRepo"
+repoUrl="https://github.com/angular/material2-docs-content"
```

Comment:
I don't think everybody can create Travis encrypted variables. I can create a small repo for it and we don't need to change the token at all. (Since its currently  running over my token)